src/corecel/data/detail/Filler.cu

```diff
@@ -5,6 +5,8 @@
 //---------------------------------------------------------------------------//
 //! \file corecel/data/detail/Filler.cu
 //---------------------------------------------------------------------------//
+#include "celeritas/Types.hh"
```

Comment:
`corecel` is upstream of `celeritas` so we unfortunately can't include it here. You should be able to make a `celeritas/track/detail/Filler.cu` that is effectively like this file but only includes instantiation for the `celeritas/Types` that you need.